### ISSUE TITLE
venv-update will install wheels that don't match the python version

### ISSUE THREAD

**Issue by asottile** [Yelp/venv-update#28]
For instance this satisfied pyscss under python2.7:

```
{'file_tags': set([(u'cp26', u'none', u'linux_x86_64')]), 'name': u'pyScss', 'abis': [u'none'], 'pyversions': [u'cp26'], 'filename': u'pyScss-1.2.0.post3-cp26-none-linux_x86_64.whl', 'version': u'1.2.0.post3', 'plats': [u'linux_x86_64']}
```

I'd test it like this:

```
1. Make a cp26 wheel under ~/.wheelhouse
2. Run venv update -ppython2.7 with that as a requirement
3. Validate that the wheel being built is in the output
4. Make sure the wheel ends up in ~/.wheelhouse after running
```

**Comment by asottile** [comment]
Update: this only happens on pypi indexes that have wheels.

I've pushed a failing test to https://github.com/Yelp/venv-update/tree/wrong_wheel

**Comment by asottile** [comment]
Fixed via #31
